docs/fundamentals/code-analysis/quality-rules/snippets/csharp/all-rules/ca2218.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+
+namespace ca2218
```

Comment:
We no longer generate this warning for C# as the C# compiler generates a CS warning for it. These two rules are now VB only in the ported analyzers.